Source/WebCore/page/Quirks.cpp

```diff
@@ -171,10 +171,7 @@ bool Quirks::needsAutoplayPlayPauseEvents() const
 // - iOS PiP
 bool Quirks::needsSeekingSupportDisabled() const
 {
-    if (!needsQuirks())
-        return false;
-
-    return isDomain("netflix.com"_s);
+    return false;
 }
```

Comment:
Can this be removed?

Reply:
+1 if it's not needed anymore, this should be removed entirely.